lib/usage.js

```diff
@@ -397,8 +397,11 @@ module.exports = function (yargs, y18n) {
 
   // guess the width of the console window, max-width 80.
   function windowWidth () {
-    const wsize = require('window-size')
-    return wsize.width ? Math.min(80, wsize.width) : null
+    if (process && process.stdout && process.stdout.columns && Math.min(80, process.stdout.colums)) {
```

Comment:
Huh, not sure what I was thinking there. What I am wondering now though, why is the output's width limited to 80 columns like that? Since `Math.min()` always returns the tiniest value of the given parameters, it can only return numbers in the range of `<=80` like that.